From: Yaniv Rosner <yanivr@broadcom.com>
Date: Sat, 28 Sep 2013 08:46:10 +0300
Subject: [PATCH 144/242] bnx2x: Change function prototype
Patch-mainline: v3.13-rc1

Git-commit: d67710ffcc37f96b34ce2af85e19600265655bb3
References: bsc#908684 FATE#317539


Change bnx2x_bsc_read function prototype (more of a cosmetic change).

Signed-off-by: Yaniv Rosner <yanivr@broadcom.com>
Signed-off-by: Yuval Mintz <yuvalmin@broadcom.com>
Signed-off-by: Eilon Greenstein <eilong@broadcom.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Cho, Yu-Chen <acho@suse.com>
---
 drivers/net/ethernet/broadcom/bnx2x/bnx2x_link.c |    5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

--- a/drivers/net/ethernet/broadcom/bnx2x/bnx2x_link.c
+++ b/drivers/net/ethernet/broadcom/bnx2x/bnx2x_link.c
@@ -3118,7 +3118,7 @@ static void bnx2x_bsc_module_sel(struct
 }
 
 static int bnx2x_bsc_read(struct link_params *params,
-			  struct bnx2x_phy *phy,
+			  struct bnx2x *bp,
 			  u8 sl_devid,
 			  u16 sl_addr,
 			  u8 lc_addr,
@@ -3127,7 +3127,6 @@ static int bnx2x_bsc_read(struct link_pa
 {
 	u32 val, i;
 	int rc = 0;
-	struct bnx2x *bp = params->bp;
 
 	if ((sl_devid != 0xa0) && (sl_devid != 0xa2)) {
 		DP(NETIF_MSG_LINK, "invalid sl_devid 0x%x\n", sl_devid);
@@ -7915,7 +7914,7 @@ static int bnx2x_warpcore_read_sfp_modul
 			usleep_range(1000, 2000);
 			bnx2x_warpcore_power_module(params, 1);
 		}
-		rc = bnx2x_bsc_read(params, phy, 0xa0, addr32, 0, byte_cnt,
+		rc = bnx2x_bsc_read(params, bp, 0xa0, addr32, 0, byte_cnt,
 				    data_array);
 	} while ((rc != 0) && (++cnt < I2C_WA_RETRY_CNT));
 
